src/overlays/actors/ovl_En_Suttari/z_en_suttari.c

```diff
@@ -748,20 +748,19 @@ s32 func_80BABF64(EnSuttari* this, GlobalContext* globalCtx, struct_80133038_arg
 }
 
 s32 func_80BABFD4(EnSuttari* this, GlobalContext* globalCtx) {
-    UNK_TYPE1 sp7C[0x424];
+    s32 pad3[9];
+    f32 sp7C[256];
```

Comment:
```suggestion
    f32 sp7C[265];
```
Should be the same as En_Pm